### ISSUE TITLE
Title theming

### ISSUE THREAD

**Issue by tomlut** [AitorDB/home-assistant-sun-card#65]
Could you please consider using the standard Home Assistant title element(s) so I don't have to create a new card-mod card class just to theme the title in your card to match everything else?

![Screenshot_2021-05-23 Overview - Home Assistant](https://user-images.githubusercontent.com/10679300/119247230-3660c980-bbcb-11eb-98b7-5da5cbd52722.png)

**Comment by AitorDB** [comment]
Yes, this should be part of the styles changes/fixes that are coming in next versions

**Comment by lanrat** [comment]
Could it also be possible to specify the colors for the SVG generated as part of a custom theme? (or card-mod)
